BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -1833,6 +1850,8 @@ void EndBoost()
 
         IEnumerator CruiseRoutine()
         {
+            float CruiseBurnRate = sustainerMass / cruiseTime; //Same as in booster
```

Comment:
If `cruiseTime` is 0, then this will become `Inf` or `NaN`. Use
```csharp
float cruiseBurnRate = (cruiseTime == 0) ? 0 : sustainerMass / cruiseTime;
```
instead. Similarly for `burnRate` above.